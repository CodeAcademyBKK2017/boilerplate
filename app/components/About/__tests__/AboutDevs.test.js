
import React from 'react';
import 'react-native';
import AboutDevs from '../AboutDevs.component';
import renderer from 'react-test-renderer';

describe('AboutDevs', () => {
  it('renders correctly', () => {
    const snapshot = renderer.create(<AboutDevs />);
    expect(snapshot).toMatchSnapshot();
  });
});