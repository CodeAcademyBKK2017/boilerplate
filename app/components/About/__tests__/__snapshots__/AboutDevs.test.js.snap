// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutDevs renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    About the Devs 
  </Text>
</View>
`;
